dashboard/tests/test_dashboard.py

```diff
@@ -47,7 +57,7 @@ def prepare_test_files():
         })
     }],
     indirect=True)
-def test_basic(ray_start_with_dashboard):
+def test_basic(disable_test_module, ray_start_with_dashboard):
```

Comment:
Why does this need to be disabled?

Reply:
Test whether the basic functions are good when the test module is disabled.